core/shell.py

```diff
@@ -1084,19 +1086,22 @@ def Main(
 
         if readline:
             term_width = 0
-            if flag.completion_display == 'nice':
-                try:
-                    term_width = libc.get_terminal_width()
-                except (IOError, OSError):  # stdin not a terminal
-                    pass
+            try:
+                term_width = libc.get_terminal_width()
+            except (IOError, OSError):  # stdin not a terminal
+                pass
```

Comment:
I think we should set term_width to DEFAULT_TERM_WIDTH here, for BOTH 'nice' and 'minimal'


Then the logic below will be cleaner

Reply:
We can also get rid of `term_width = 0` then, since it will be initialized in both cases